.github/workflows/build_shadow.yml

```diff
@@ -43,50 +51,82 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - name: Set compiler params
+        run: |
+          # See https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions
+          # for how these 'echo' commands work.
+          case "${{ matrix.cc }}" in
+            "gcc")
+              echo '::set-env name=CC:gcc'
```

Comment:
Hi @stevenengler - I got the idea to use workflow commands (which I previously didn't know about) from how you added rust to the path, below. These seem to be having no effect, though. Any idea what might be going wrong?

Reply:
It looks okay to me, but maybe try with for example `CC::gcc` instead of `CC:gcc`. If that doesn't work, maybe you can only have one command per shell? Hopefully not, but other than that I don't have any ideas.

Reply:
> `CC::gcc` instead of `CC:gcc`

:facepalm: - that was it. Thanks for spotting it! :)

Reply:
No problem, glad it was a simple change :)